scripts/code_generation/jsoncan/src/codegen/dbc_generation/dbc_generation.py

```diff
@@ -34,11 +34,9 @@
 )
 DBC_VALUE_TABLE_TEMPLATE = "VAL_ {id} {signal_name} {entries};\n"
 
-
-# BA_DEF_ BO_  "GenMsgCycleTime" INT {cycle_time_min} {cycle_time_max};
-# BA_DEF_ SG_  "GenSigStartValue" INT {start_value_min} {start_value_max};
-# BA_DEF_DEF_  "GenMsgCycleTime" {cycle_time_default};
-# BA_DEF_DEF_  "GenSigStartValue" {start_value_default};
+# If a signal isn't recevied by any node then DBC parsing might fail.
+# Hacky solution: Every signal receives "DEBUG"
+DBC_DEFAULT_RECEIVER = "DEBUG"
```

Comment:
can you not call this debug as to differentiate with our debug node?

Reply:
I think my idea is that these are the same - "DEBUG" is essentially someone with PCAN so it makes sense that the person doing the monitoring is receiving them all